src/app/components/TradingChart/index.tsx

```diff
@@ -89,18 +98,25 @@ export function TradingChart(props: ChartContainerProps) {
 
     // run only once after mount
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [client]);
 
   useLayoutEffect(() => {
     if (chart && hasCharts) {
+      chart.chart().resetData();
+
+      // if quote asset is not RBTC or XUSD, make it line chart, otherwise candle
+      // 1 - candles
+      // 2 - line
+      chart.chart().setChartType(hasDirectFeed(props.symbol) ? 1 : 2);
```

Comment:
Make this an enum (there is `SeriesStyle` enum in `charting_library` with all of the values).

Reply:
Using this type crashes build because of typescript configs, fixing config breaks another part of the dapp. I dont think it's worth debugging just for having typing there

Reply:
Yeah I know you cannot use `SeriesStyle` enum but you can basically copy it to our solution and use it.

Reply:
Adding, but re-casting to a number will be needed, because type in the setChartType property will not match, even if values are exactly the same.